LLMListView/SwipeReleaseAnimationConstructor.cs

```diff
@@ -182,7 +182,6 @@ public void SwipeTo(SwipeDirection direction, SwipeConfig config, bool animated)
             else
             {
                 config.MainTransform.X = targetX;
```

Comment:
@F1nZeR  Curiously, I also don't know why. Only visible item's transform will be reseted to 0, you can find the other items is ok if you scroll the list. 
ContentPresenterTranslateTransform is origin ListViewItem style property, it seems that ContentPresenterTranslateTransform is changed when render the item.